ios/InstagramFirebase/CommentInputTextView.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  CommentInputTextView.swift
+//  InstagramFirebase
+//
+//  Created by Brian Voong on 1/3/18.
+//  Copyright © 2018 Lets Build That App. All rights reserved.
+//
+
+import UIKit
+
+class CommentInputTextView: UITextView {
+    
+    fileprivate let placeholderLabel: UILabel = {
```

Comment:
```suggestion
    fileprivate let placeholderLabel: UILabel = {
    let labe = UILabel 
    return label
    }()
```

**Apply** for all UI components.